.vscode/settings.json

```diff
@@ -6,8 +6,11 @@
         }
     ],
     "eslint.options": {
-        "rulePaths": ["../scripts/eslint/built/rules/"],
+        "rulePaths": ["./scripts/eslint/built/rules/"],
         "ext": [".ts"]
     },
-    "eslint.workingDirectories": ["./src", "./scripts"]
+    "eslint.workingDirectories": [
```

Comment:
Is this options required for this repo?
I tried to remove it, used `"rulePaths": ["./scripts/eslint/built/rules/"]` and everything looks fine at first glance (for user who don't know what rules should be applied where):
 - `space-in-parens`, `@typescript-eslint/semi`, and custom `boolean-trivia` work in both `src` and `scripts` folders
 - `no-var` suppressed in `src\lib\esnext.bigint.d.ts`, but will be reported as error if I comment out supression.
 - no `ENOENT` errors when I open files in `tests` folder

Reply:
You're right - since we only have one ESLint config file, we shouldn't need this.
Works well on my box as you suggest.

Reply:
To be fair - there is second `src/.eslintrc.json` config (and third in same folder), but I tried to tweak it (commented out `no-var` suppression) and saw expected results. Looks like `files` in overrides were resolved as patterns relative to config anyway.

Reply:
Cool, that works for me too.